src/Juvix/Compiler/Pipeline.hs

```diff
@@ -89,6 +89,14 @@ upToCore ::
 upToCore =
   upToInternalReachability >>= Core.fromInternal
 
+--- | A version of upToCore that does not filter out unreachable symbols.
+--- | Intended for use with the REPL.
+upToCoreRepl ::
```

Comment:
have you considered adding a field to the entrypoint that indicates if we want to skip reachability filtering? it seems to be slightly better since then we don't need to add another pipeline segment specific to the repl.

Reply:
Good idea - I'll do it this way

Reply:
Done

Reply:
I completely misread your comment, I've now added a field to the entrypoint record.